gitcha.go

```diff
@@ -86,6 +110,29 @@ func FindFiles(path string, list []string) (chan SearchResult, error) {
 				}
 			}
 
+			for _, pattern := range ignorePatterns {
+				// If there's no path separator in the pattern try to match
+				// against the directory we're currently walking.
+				if !strings.Contains(pattern, string(os.PathSeparator)) {
+					dir := filepath.Dir(path)
+					if dir == "." {
+						continue // path is empty
+					}
+					pattern = dir + string(os.PathSeparator) + pattern
```

Comment:
We should use `filepath.Join` here.

Reply:
Good call. Updated in the latest revision.